.vsts-ci.yml

```diff
@@ -291,6 +291,19 @@ extends:
               publishArgument: $(_publishArgument)
               officialBuildProperties: $(_officialBuildProperties)
               runTests: false
+      ### ARM64 TESTBUILD ###
+      - ${{ if or(eq(parameters.runTestBuild, true), eq(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
This will only run on internal PRs or when `runTestBuild` checkbox is checked when running the build manually.